### ISSUE TITLE
Shortcuts: module reset "right-activate" is not explained

### ISSUE THREAD

**Issue by grothesque** [darktable-org/dtdocs#533]
(Moved over from https://github.com/darktable-org/darktable/issues/13051.)

This applies to the current development version of darktable, but I assume that it applies to 4.0 and other versions just as well.

When defining a "reset" shortcut for a module, there are three possible choices for "effect":
- activate
- ctrl-activate
- right-activate

The first two possibilities are explained in https://docs.darktable.org/usermanual/4.0/en/preferences-settings/shortcuts/#utility-modules, but what does the third one do?  Is it equivalent to right-clicking on the reset button of the module?  But what does the latter do?  This is not explained in https://docs.darktable.org/usermanual/4.0/en/darkroom/processing-modules/module-header/.

Searching the entire documentation for "right-activate" or even "right" does not seem to yield an answer as well.

It seems to me that the documentation is incomplete in this regard.

**Comment by elstoc** [comment]
Unfortunately the keyboard shortcut functionality is so complex and multi-functional that describing everything it can do would make the documentation complex, unwieldy and hard to follow for a new user (I'm honestly not happy with how complex it is at the moment). The documentation is therefore, by necessity, fairly high level, with the expectation being that the user will have do do some experimentation.

Regarding your particular query, the shortcut functionality is fairly generic, treating UI elements as "buttons" or "sliders" or "toggles" etc. It doesn't care what functionality is _available_ on a particular UI element (like the "button" that is used to reset a module), but it just tries to simulate a generic set of possible user actions against a generic "button". When you create a "button" in the code, the creation of that button automatically generates the generic list of button shortcuts. On some buttons right-click (which I assume is what "right-activate" means in this context) does nothing. Equally on other buttons ctrl-activate (ctrl+click) does nothing. I think "Activate" is just the Gtk term for clicking a button (activate because it can be activated by other means as well as a mouse-click).

The documentation does state (under the "common actions" section) that "it is possible to assign a number of actions that have no effect. For example, all sliders include a button element, regardless of whether such a button is actually present alongside a given slider" which is my attempt to convey the generic nature of the shortcut system.

TLDR: I agree that the documentation is incomplete, but I think making it complete would be to the detriment of readability and usefulness.
